tests/queryApiTest/chainAndAccountInfo.js

```diff
@@ -0,0 +1,37 @@
+const assert = require('chai').assert;
+const helper = require('../helper.js');
+const accounts = helper.ACCOUNTS;
+
+describe('Query api calls:', async () => {
+  let api,user;
+ 
+
+  before(async () => {
+    const avnApi = await helper.avnApi({
+      suri: accounts.user.seed
+    });
+    api = await avnApi.apis();
+    user = accounts.user;
+  });
+
+
+ describe('getChainInfo', async () => {
+    it('@NO_BASELINE can get the current chain information', async () => {
```

Comment:
I think we could remove the `@NO_BASELINE` if we are not planning on having baseline tests.